content/applications/sales/sales/amazon_connector/setup.rst

```diff
@@ -1,104 +1,151 @@
+#. Go to :menuselection:`Sales app --> Configuration --> Settings --> Connectors --> Amazon Sync -->
    Amazon Marketplaces`.
-#. Create a new marketplace record. Enter the :guilabel:`Marketplace ID` and select the
-   :guilabel:`Amazon Region` for your marketplace as described in the `Amazon Documentation for
-   marketplace IDs and regions
-   <https://developer-docs.amazon.com/amazon-shipping/docs/marketplace-ids>`_, and the
+#. Click :guilabel:`New` to create a new marketplace record.
+#. Enter the Marketplace ID in the :guilabel:`API Idenifier` field, and select the :guilabel:`Amazon
+   Region` for your marketplace as described in the `Amazon Documentation for marketplace IDs and
+   regions <https://developer-docs.amazon.com/amazon-shipping/docs/marketplace-ids>`_, and the
    :guilabel:`Seller Central URL` as described in the `Amazon Documentation for seller central URLs
    <https://developer-docs.amazon.com/amazon-shipping/docs/seller-central-urls>`_.
-#. Set the name of the record to `Amazon.<country code>` to easily retrieve it (e.g.:
-   **Amazon.se**). The :guilabel:`API Identifier`, the :guilabel:`Region` and the :guilabel:`Seller
-   Central URL` fields should respectively hold the *MarketplaceId*, the selected Amazon region and
-   the *Seller Central URL* values from the Amazon Documentation.
+#. Set the name of the record to `Amazon.<country code>` to easily retrieve it (e.g.
+   `Amazon.se`). The :guilabel:`API Identifier`, the :guilabel:`Region` and the :guilabel:`Seller
+   Central URL` fields should respectively hold the *Marketplace ID*, the selected Amazon region,
+   and the *Seller Central URL* values from the Amazon Documentation.
 #. Once the marketplace is saved, update the Amazon Account configuration by going to
-   :menuselection:`Sales --> Configuration --> Settings --> Connectors --> Amazon Sync -->
-   Amazon Accounts`. Open the account on which you wish to use the new marketplace, go to the
-   :guilabel:`Marketplaces` tab and click on :guilabel:`Update available marketplaces` (an animation
-   should confirm the success of the operation). Newly added marketplaces are automatically added to
-   the list of synchronized marketplaces. If the new marketplace is not added to the list, it means
-   that it is either incompatible or unavailable for your seller account.
+   :menuselection:`Sales app --> Configuration --> Settings --> Connectors --> Amazon Sync -->
+   Amazon Accounts`.
+#. Select the account on which to use the new marketplace, go to the :guilabel:`Marketplaces` tab,
+   and click on :guilabel:`Update available marketplaces` (an animation should confirm the success
+   of the operation). Newly added marketplaces are automatically added to the list of synchronized
+   marketplaces. If the new marketplace is not added to the list, it means that it is either
+   incompatible or unavailable for the seller account.
```

Comment:
Any alternative solutions in this case, if someone runs into this?

Reply:
Again, without proper access to a paid-for Amazon Seller account to create this documentation, the depth of topics is a bit limited. So, if they run into an error at this point, they'll have to reach out to support or figure out a solution on their own. That said, if we get comments requesting such a solution, we can reach out to POs for more info. But, I believe, for the time being, this can stand as-is. Thanks for pointing it out, though.